IPython/lib/pretty.py

```diff
@@ -196,19 +196,22 @@ def __init__(self, output, max_width=79, newline='\n', max_seq_length=MAX_SEQ_LE
         self.group_queue = GroupQueue(root_group)
         self.indentation = 0
 
-    def _break_outer_groups(self):
+    def _break_one_group(self, group):
+        while group.breakables:
+            x = self.buffer.popleft()
+            self.output_width = x.output(self.output, self.output_width)
+            self.buffer_width -= x.width
+        while self.buffer and isinstance(self.buffer[0], Text):
+            x = self.buffer.popleft()
+            self.output_width = x.output(self.output, self.output_width)
+            self.buffer_width -= x.width
+
+    def _break_outer_groups(self, force=):
```

Comment:
You appear to have a syntax error here, and `force` seem unused. Did you forgot some commits ?

Reply:
```suggestion
    def _break_outer_groups(self):
```

Nope, just clumsy in-browser editing